test_scripts/API/VehicleData/StabilityControlsStatus/019_UnsubscribeVehicleData_StabilityControlsStatus_disallowed_by_policy.lua

```diff
@@ -0,0 +1,36 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: UnsubscribeVehicleData RPC with `stabilityControlsStatus` parameter only is NOT allowed by Policies
```

Comment:
@LitvinenkoIra Unclear description. 
- UnsubscribeVehicleData RPC with the only parameter `stabilityControlsStatus`, which is NOT allowed by Policies

Reply:
@AByzhynar Done in 4689ec2